gdpr/migrations/0004.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.16 on 2018-12-19 15:17
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('gdpr', '0003'),
+    ]
+
+    operations = [
+        migrations.AlterUniqueTogether(
```

Comment:
Tohle jsem musel vytáhnout ven, protože to padalo na tom že změna schamatu nejde dělat s datovou změnou. Jak propaguju vše v jedné migraci, tak toto se mi stalo poprvé a docela mě to překvapilo :-/